.github/workflows/terraform.yml

```diff
@@ -15,7 +19,7 @@ jobs:
 
       - name: Install Terraform
         uses: hashicorp/setup-terraform@v1
-        with: { terraform_version: 0.13.0 }
+        with: { terraform_version: 0.14.10 }
```

Comment:
I removed this completely for now.

Reply:
If the you remove the version constraint it will per default install the latest version.

Reply:
That's fair enough. Do you mind updating this to the latest version as of this moment?

Reply:
The code is not updated to the latest version yet so i recommend leaving that as is until we update the code in another PR

Reply:
I have made updates in https://github.com/telia-oss/terraform-aws-jaeger/pull/4 or do you mean something different?

Reply:
If we specify terraform version, then we get deterministic builds. The build is green for a specific version of Terraform. I think this is a good thing.

If we don't, then if somebody later makes a PR and the build fails, the PR could have failed because of some other reason than the new code being wrong. It can be caused by a non backwards compatible Terraform upgrade.

We specify the terraform version in almost all of the other telia-oss Actions builds, I believe.